ocaml/src/gobjfs/gobjfs_test.ml

```diff
@@ -0,0 +1,121 @@
+(*
+Copyright (C) 2016 iNuron NV
+
+This file is part of Open vStorage Open Source Edition (OSE), as available from
+
+
+    http://www.openvstorage.org and
+    http://www.openvstorage.com.
+
+This file is free software; you can redistribute it and/or modify it
+under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+as published by the Free Software Foundation, in version 3 as it comes
+in the <LICENSE.txt> file of the Open vStorage OSE distribution.
+
+Open vStorage is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY of any kind.
+*)
+
+open Gobjfs
+open Lwt.Infix
+
+let fill data=
```

Comment:
`ocaml/src/gblob_access_test.ml` also has an almost identical copy of the `fill` and `check` functions that are present in this file.
They differ in the constant used for the modulo, one uses 256 and the other 100. I would recommend 251 because it's the biggest prime number smaller than 256.